group_vars/matrix_servers

```diff
@@ -448,41 +448,45 @@ matrix_mailer_enabled: true
 
 ######################################################################
 #
-# matrix-mxisd
+# matrix-ma1sd
 #
 ######################################################################
 
-# By default, this playbook installs the mxisd identity server on the same domain as Synapse (`matrix_server_fqn_matrix`).
+# By default, this playbook installs the ma1sd identity server on the same domain as Synapse (`matrix_server_fqn_matrix`).
 # If you wish to use the public identity servers (matrix.org, vector.im) instead of your own you may wish to disable this.
-matrix_mxisd_enabled: true
+matrix_ma1sd_enabled: true
 
+<<<<<<< HEAD
```

Comment:
Those merge comments should be removed 😄

Reply:
oh good catch, how did that happen :sweat_smile: